cli/src/pcluster/api/util.py

```diff
@@ -164,3 +172,31 @@ def assert_node_executable():
         )
         LOGGER.critical(message)
         raise Exception(message)
+
+
+def assert_node_version():
+    try:
+        node_version = subprocess.check_output(["node", "--version"], stderr=subprocess.STDOUT, text=True)
+        LOGGER.debug("Found nodejs version (%s)", node_version)
```

Comment:
Did you check which is our default log level for the cli? It would be good to have this information always available so eventually moving it to info.

Reply:
Minor: `Found Node.js ...`

Reply:
default is info, switched log level to info